examples/angular_bazel_architect/projects/frontend-lib/karma.conf.js

```diff
@@ -28,7 +31,23 @@ module.exports = function (config) {
     customLaunchers: {
       ChromeHeadlessNoSandbox: {
         base: 'ChromeHeadless',
-        flags: ['--no-sandbox', '--disable-gpu', '--disable-dev-shm-usage']
+        // `--no-sandbox` flag disables the chrome sandbox because it causes Chrome to crash on some
```

Comment:
should we do sth about the code duplication here? we could generate karma.conf but maybe that makes it too different from standard angular?